src/core/thread/mle_constants.hpp

```diff
@@ -105,6 +105,18 @@ enum
     kMleEndDeviceTimeout        = OPENTHREAD_CONFIG_DEFAULT_CHILD_TIMEOUT,  ///< MLE_END_DEVICE_TIMEOUT (seconds)
 };
 
+/**
+ * Parent Priority values
+ *
+ */
+enum
+{
+    kParentPriorityHigh     = 1,    // Parent Priority High
+    kParentPriorityMedium   = 0,    // Parent Priority Medium (default)
+    kParentPriorityLow      = -1,   // Parent Priority Low
+    kParentPriorityReserved = -2,   // Parent Priority Reserved
```

Comment:
Updated. The value is defined as "reserved" in Spec, but I agree that using "unspecified" here is more readable.